flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -133,8 +136,8 @@ public void testChainStartEndSetting() throws Exception {
 				public Integer map(Integer value) throws Exception {
 					return value;
 				}
-			})
-			.print();
+			}).setParallelism(2)
```

Comment:
Is this change needed? It seems to be an unrelated test.

Reply:
It's an unrelated change in a sense.
I found it when adding unit test. It's a potential unstable case since it depends on the `StreamExecutionEnvironment.defaultLocalParallelism` which must be not 1 in this case.
However I just found someone has fix it yesterday. What a coincident! I can remove it from this PR.